packages/core/src/libraries/jwt-customizer.ts

```diff
@@ -1,14 +1,18 @@
+import { runCustomJwtClaimsScriptInLocalVm, buildErrorResponse } from '@logto/azure-functions-kit';
```

Comment:
nit: is it necessary to create another package? The name "VM" should not depend on "azure functions".

Reply:
not a nit imo